tests/util/test_sha_utils.py

```diff
@@ -14,3 +14,12 @@ def test_sha_string(self):
             SHAUtils.sha_string(""),
             "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
         )
+
+    def test_sha_file_at_path(self):
+        self.assertEqual(
+            # 6.3 KiB file of random base64 data and new lines
+            SHAUtils.sha_file_at_path("tests/util/resources/sha_test_data.txt", read_block_size=2048),
```

Comment:
What happens if the read_block_size is set wrong - e.g. if we used the default here?

Reply:
You can't set it wrong. It's set low here so I can have a sensible default without needing to commit 100s of 1000s of random characters to the git repo. I could add a test which uses the default though.